dunecore/Geometry/gdml/protodunevd_v4_refactored.gdml

```diff
@@ -40638,6 +40650,32 @@
       <materialref ref="STEEL_STAINLESS_Fe7Cr2Ni" />
       <solidref ref="SteelShell" />
     </volume>
+    
+    <!-- Anode -->
+    
+    <volume name="volAnode_nonTCOSide_bot">
```

Comment:
Why do we need to especify the 4 different anodes? Can't they just be replicated? The same thing for the two cathodes